example/src/main/java/org/apache/rocketmq/example/ordermessage/Producer.java

```diff
@@ -52,9 +52,8 @@ public MessageQueue select(List<MessageQueue> mqs, Message msg, Object arg) {
 
                 System.out.printf("%s%n", sendResult);
             }
-
             producer.shutdown();
-        } catch (MQClientException | RemotingException | MQBrokerException | InterruptedException e) {
+        } catch (MQClientException | RemotingException | MQBrokerException | InterruptedException | UnsupportedEncodingException e) {
```

Comment:
How about  `catch(Exception e)` ?

Reply:
Just with  e.printStackTrace();

Reply:
catch (Exception e) {
            e.printStackTrace();
        }

Reply:
copied that.

Reply:
So please review my new committed code.

Reply:
Please review final code committed.